nft_ingester/src/program_transformers/token/mod.rs

```diff
@@ -72,9 +72,12 @@ pub async fn handle_token_program_account<'a, 'b, 'c>(
                 .one(&txn)
                 .await?;
             if let Some(asset) = asset_update {
-                let mut active: asset::ActiveModel = asset.into();
-                active.owner = Set(Some(owner));
-                active.save(&txn).await?;
+                // will only update owner if token account balance is non-zero
+                if ta.amount > 0 {
```

Comment:
Would it make sense to leverage the "slot_updated" field here? I'm concerned these changes will miss edge cases.